provider/pkg/schema/gen.go

```diff
@@ -936,6 +942,22 @@ func (ctx *context) propertySpec(propName, resourceTypeName string, spec *jssche
 		}
 	}
 
+	if propName == "Tags" {
+		switch ctx.GetTagsStyle(typeSpec, spec) {
+		case TagsStyleUntyped:
+			if propertySpec.Description != "" {
+				propertySpec.Description += "\n\n"
+			}
+			propertySpec.Description += "No additional type information is available in the schema for this property, please refer to the Cloud Control documentation for the expected schema."
+		case TagsStyleStringMap:
+			// Nothing to do
```

Comment:
So IIUC, the idea is that these `map[string]string` shaped Tag properties will just stay `map[string]string`, but in the next review, we'll start converting things that are a list of `{ key:string, value: string }` to be `[]Tag` rather than `[]SomeResourceTag`

Reply:
Nope, in the design discussion Luke pushed back strongly on unifying tag shapes and to just leave them as-is .. so we're only reducing the duplication of types now.